bundles/org.openhab.binding.volvooncall/.classpath

```diff
@@ -28,5 +28,14 @@
 			<attribute name="maven.pomderived" value="true"/>
 		</attributes>
 	</classpathentry>
+	<classpathentry kind="lib" path="/Users/Kjoglum/.m2/repository/org/openhab/core/bundles/org.openhab.core.thing/2.5.0-SNAPSHOT/org.openhab.core.thing-2.5.0-SNAPSHOT.jar" sourcepath="/Users/Kjoglum/.m2/repository/org/openhab/core/bundles/org.openhab.core.thing/2.5.0-SNAPSHOT/org.openhab.core.thing-2.5.0-SNAPSHOT-sources.jar">
```

Comment:
1°/ This classpathentry countains local path definition.
2°/ This package is already included in the build so it does not have to be specifically included.